src/Kart/Leaflet.js

```diff
@@ -25,34 +25,34 @@ class Leaflet extends React.Component {
     clickCoordinates: { x: 0, y: 0 },
     markerType: "klikk",
     showInfobox: false,
-    coordinates_area: null,
+    coordinates_area: null
```

Comment:
Alle disse endringene har kommet av @bjornreppen sin autoformatterer, så det er mulig dere har forskjellig instillinger her. En viss fare for at de legges til igjen automatsk :)